compiler/testSuite/renamings.txt

```diff
@@ -0,0 +1,497 @@
+Vfloat05             -> 06-float/12-v.bal
+Efloat06             -> 06-float/13-e.bal
+Vfloat06             -> 06-float/14-v.bal
+Efloat07             -> 06-float/15-e.bal
+Vfloat07             -> 06-float/16-v.bal
+Efloat08             -> 06-float/17-e.bal
+Vfloat08             -> 06-float/18-v.bal
+Vfloat09             -> 06-float/19-v.bal
+Vfloat10             -> 06-float/20-v.bal
+Vfloat11             -> 06-float/21-v.bal
+Vfloat12             -> 06-float/22-v.bal
+Vfloat13             -> 06-float/23-v.bal
+Vfloat14             -> 06-float/24-v.bal
+Vconstfloat00        -> 06-float/const1-v.bal
+Vconstfloat01        -> 06-float/const2-v.bal
+Vconstfloat02        -> 06-float/const3-v.bal
+Vnarrow07            -> 06-narrow/01-v.bal
+Enarrow08            -> 06-narrow/02-e.bal
+Vnarrow08            -> 06-narrow/03-v.bal
+Enarrow09            -> 06-narrow/04-e.bal
+Vnarrow09            -> 06-narrow/05-v.bal
+Enarrow10            -> 06-narrow/06-e.bal
+Vnarrow10            -> 06-narrow/07-v.bal
+Enarrow11            -> 06-narrow/08-e.bal
+Vnarrow11            -> 06-narrow/09-v.bal
+Enarrow12            -> 06-narrow/10-e.bal
+Enarrow13            -> 06-narrow/11-e.bal
+Vunreach00           -> 06-narrow/unreach-v.bal
+Voptional00          -> 06-optional/1-v.bal
+Voptional01          -> 06-optional/2-v.bal
+Ptypecast01          -> 06-typecast/01-p.bal
+Ptypecast02          -> 06-typecast/02-p.bal
+Ptypecast03          -> 06-typecast/03-p.bal
+Ptypecast04          -> 06-typecast/04-p.bal
+Etypecast05          -> 06-typecast/05-e.bal
+Ptypecast05          -> 06-typecast/06-p.bal
+Vtypecast05          -> 06-typecast/07-v.bal
+Etypecast06          -> 06-typecast/08-e.bal
+Vtypecast06          -> 06-typecast/09-v.bal
+Vtypecast07          -> 06-typecast/10-v.bal
+Vtypecast08          -> 06-typecast/11-v.bal
+Vtypecast09          -> 06-typecast/12-v.bal
+Vtypecast10          -> 06-typecast/13-v.bal
+Vtypecast11          -> 06-typecast/14-v.bal
+Vtypecast12          -> 06-typecast/15-v.bal
+Vtypecast13          -> 06-typecast/16-v.bal
+Vtypecast14          -> 06-typecast/17-v.bal
+Vtypetest03          -> 06-typetest/1-v.bal
+Vtypetestnnot00      -> 06-typetest/nnot-v.bal
+Etypetestnot00       -> 06-typetest/not1-e.bal
+Etypetestnot01       -> 06-typetest/not2-e.bal
+Vtypetestnot01       -> 06-typetest/not3-v.bal
+Vunion00             -> 06-union/1-v.bal
+Vbenchintmergesort   -> 07-bench/intmergesort-v.bal
+Eerror00             -> 07-error/1-e.bal
+Verror00             -> 07-error/2-v.bal
+Eerror01             -> 07-error/3-e.bal
+Verror01             -> 07-error/4-v.bal
+Verror02             -> 07-error/5-v.bal
+Verror03             -> 07-error/6-v.bal
+Verror04             -> 07-error/7-v.bal
+Verror05             -> 07-error/8-v.bal
+Verror06             -> 07-error/9-v.bal
+Ppanic00             -> 07-error/panic1-p.bal
+Ppanic01             -> 07-error/panic2-p.bal
+Ppanic02             -> 07-error/panic3-p.bal
+Plist05              -> 07-list/1-p.bal
```

Comment:
07-array is better

array goes with map; list goes with mapping

Reply:
Ok, let's do the individual improvements later, but if you notice any renameings that involve multiple files, it might be easier to modify the script now, please let me know if you notice such cases.

I will work on modifying `test/Makefile`.
I'll rename `new.mk`` and `sub.mk` to `balt.mk` and `balt-sub.mk`.
Then I can introduce a new `sub.mk` and invoke it form `test/Makefile`.